### PR TITLE
Update Dataset.md to add license restrictions

### PR DESCRIPTION
**Pull request by rgopikrishnan91**
@kestewart Please review if the added constraints look okay. This is in accordance with my best interpretation of #462

### CODE REVIEW

File: model/Dataset/Classes/Dataset.md [hunk 1]
```diff
@@ -72,9 +72,15 @@ Metadata information that can be added to a dataset that may be used in a softwa
   - minCount: 1
 - /Software/Package/downloadLocation
   - minCount: 1
+ 
+## External relationship restrictions
+- relationshipType: declaredLicense
+  - fromType: /Software/Package/Dataset
+- relationshipType: concludedLicense
+  - fromType: /Software/Package/Dataset
 - /Software/SoftwareArtifact/primaryPurpose
```
Comment (maxhbr) [inline model/Dataset/Classes/Dataset.md:81]: this should be in `## External properties restrictions` ?

**Review by maxhbr — CHANGES_REQUESTED** [review]

**Review by zvr — CHANGES_REQUESTED** [review]
Please use the format of #524 to express restrictions in a separate file,

**Comment by rgopikrishnan91** [comment]
@zvr Can you please review if this is conformant now?

**Comment by rgopikrishnan91** [comment]
Added the necessary changes to #536 . So closing this seperate PR

**Pull request closed without merging**